src/nupic/regions/CLAClassifierRegion.py

```diff
@@ -240,11 +240,11 @@ def __init__(self,
     self.recordNum = 0
     self._initEphemerals()
 
-    # Flag to know if the compute() function is ever called. This is to 
+    # Flag to know if the compute() function is ever called. This is to
     # prevent backward compatibilities issues with the customCompute() method
-    # being called at the same time as the compute() method. Only compute() 
-    # should be called via network.run(). This flag will be removed once we 
-    # get to cleaning up the clamodel.py file.
+    # being called at the same time as the compute() method. Only compute()
+    # should be called via network.run(). This flag will be removed once we
+    # get to cleaning up the htmpredictionmodel.py file.
```

Comment:
I was just following previous convention. Should I change both the module name and file name?